config/kubernetes-sigs/sig-release/teams.yaml

```diff
@@ -1,17 +1,21 @@
 teams:
   k8s-container-image-promoter-admins:
-    description: "admin access to k8s-container-image-promoter"
+    description: admin access to k8s-container-image-promoter
```

Comment:
One comment: While it is up to sig-release to administer their repos as they see fit, I think it's an anti-pattern to tie sig chairpersonship to a repos permission set.

Chairs are a process role, as defined in the dig governance docs. There are certain places where it makes sense for the chairs/leads to automatically be in owners files (mainly k/community meta docs, and k/org github meta data -- specifically where being in the owners file is directly tied to administering the sig), but I think in most cases actual access to things should be delegated to subproject owners.

This is similar to how the steering committee no longer gets automatic github owner to everything, rather they delegate that to the subproject that administers it.

To be clear though, this is just my opinion, and is not blocking (strictly speaking)

cc: @kubernetes/sig-release

Reply:
@cblecker -- That's fair. 

To be more explicit, SIG Release Chairs are [subproject owners](https://github.com/kubernetes/sig-release/blob/cdbd4045aded4a6c45ec81b308f7f8de07ec5eee/OWNERS_ALIASES#L11-L19) of the Release Engineering project and [should have ownership of all tools/processes which have the potential to affect a Kubernetes release](https://github.com/kubernetes/sig-release/issues/732).

I was going to cite some things from our charter, specifically, [deviations from SIG governance](https://github.com/kubernetes/community/blob/master/sig-release/charter.md#deviations-from-sig-governance) e.g., SIG Release does not have Technical Leads, as the Chairs are required to be technical, but it seems the charter is [due for some updates](https://github.com/kubernetes/sig-release/issues/378) (which I'll action on this cycle).

p.s. agreed on the Steering point; I did open [the issue](https://github.com/kubernetes/org/issues/637), after all. :stuck_out_tongue_closed_eyes:

Reply:
Ahh okay. I think we then just need to be explicit that Stephen and Caleb are added to this group because they are subproject owners, *not* because they are chairs. They're two distinct jobs, even though the same person can hold both. Is that right?

Reply:
@cblecker -- yep, yep. Updated the PR description and commit comment to reflect.